### ISSUE TITLE
Add glyph for R magrittr pipe %>%

### ISSUE THREAD

**Issue by jmpanfil** [microsoft/cascadia-code#653]
Would it be possible to get a glyph for the `magrittr` pipe `%>%` in RStudio? The default pipe `|>` glyph looks great but I still use `%>%` pretty often in my code.

[FiraCode](https://github.com/tonsky/FiraCode/issues/473) has a few possibilities in their never ending 5 year thread on the subject. I have no opinion on what the glyph should look like, honestly. Just any glyph would be great :)

**Comment by adRn-s** [comment]
Agreed, there are a lot of infix operators that use the between percentage-signs in R (e.g. `%in%`. Users can even define their own (and package it, like magrittr did). 👍🏽
